data-model/fdo-types/0.3.0/digital-specimens/schema/chronometric-age.json

```diff
@@ -0,0 +1,125 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/fdo-types/0.3.0/digital-specimens/chronometric-age.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$comment": "DigitalObject Version 0.3.0",
+  "type": "object",
+  "properties": {
+    "chrono:verbatimChronometricAge": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/chrono/terms/verbatimChronometricAge",
+      "examples": [
+        "27 BC to 14 AD"
+      ]
+    },
+    "ods:verbatimChronometricAgeProtocol": {
+      "type": "string",
+      "$comment": "Unknown term"
+    },
+    "chrono:uncalibratedChronometricAge": {
```

Comment:
what namespace is this?

Reply:
The is the chronometric age extension on Darwin Core
https://chrono.tdwg.org/